pyne/mesh.py

```diff
@@ -1252,23 +1392,86 @@ def tag_cell_fracs(self, cell_fracs):
             cell_largest_frac_number[i] = \
                     int(voxel_cell_number[i, largest_index])
 
+
         # create the tags
         self.tag(name='cell_number', value=voxel_cell_number,
                  doc='cell numbers of the voxel, -1 used to fill vacancy',
-                 tagtype=IMeshTag, size=max_num_cells, dtype=int)
+                 tagtype=NativeMeshTag, size=max_num_cells, dtype=int)
         self.tag(name='cell_fracs', value=voxel_cell_fracs,
-                 tagtype=IMeshTag, doc='volume fractions of each cell in the '
+                 tagtype=NativeMeshTag, doc='volume fractions of each cell in the '
                                        'voxel, 0.0 used to fill vacancy',
                  size=max_num_cells, dtype=float)
         self.tag(name='cell_largest_frac_number',
-                 value=cell_largest_frac_number, tagtype=IMeshTag,
+                 value=cell_largest_frac_number, tagtype=NativeMeshTag,
                  doc='cell number of the cell with largest volume fraction in '
                      'the voxel', size=1, dtype=int)
         self.tag(name='cell_largest_frac', value=cell_largest_frac,
-                 tagtype=IMeshTag, doc='cell fraction of the cell with largest'
-                                       'cell volume fraction',
+                 tagtype=NativeMeshTag, doc='cell fraction of the cell with '
+                                            'largest cell volume fraction',
                  size=1, dtype=float)
 
+class StatMesh(Mesh):
+    """This class extends the basic Mesh class by modifying the standard
+    mathematical operations that are performed on multiple meshes.
+
+    A StatMesh assumes that each value being operated upon also has a
+    statistical error associaed with it, and forces operations on the
+    statistical error as well.  For any tag with name `tag_name` the StatMesh
+    assumes that there is also a tag with name `tag_name_rel_error`.
+
+    For example, when to quantities are added together, c = a + b, the
+    statistical error of c is found by combining the statistical errors of a
+    and b.
+
```

Comment:
Unnecessary blank line (see pep8 and pep257)
```suggestion
```

PEP257 recommends one blank line after a class docstring. 

> Insert a blank line after all docstrings (one-line or multi-line) that document a class -- generally speaking, the class's methods are separated from each other by a single blank line, and the docstring needs to be offset from the first method by a blank line.